torch/cuda/_memory_viz.py

```diff
@@ -89,7 +89,9 @@ def _block_extra(b):
 
 def format_flamegraph(flamegraph_lines, flamegraph_script=None):
     if flamegraph_script is None:
-        flamegraph_script = f"/tmp/{os.getuid()}_flamegraph.pl"
+        cache_dir = os.path.expanduser("~/.cache/")
```

Comment:
I'm late to this game but this folder contains a lot of things on many machines, can we use a `pytorch` subfolder in there to help orgaize things?